diofant/tests/concrete/test_sums_products.py

```diff
@@ -925,8 +925,10 @@ def test_sympyissue_21557():
 def test_sympyissue_21651():
     a = Sum(floor(2*2**(-n)), (n, 1, 2))
     b = floor(2*2**(-1)) + floor(2*2**(-2))
-
     assert a.doit() == b.doit()
+    c = Sum(ceiling(2*2**(-n)), (n, 1, 2))
```

Comment:
Also I suggest remove redundant parentheses around the exponent here and below.

Reply:
Fair point